problemsets-mat223/modules/module7.tex

```diff
@@ -44,7 +44,22 @@
 	\end{equation}
 	Rewrite each system as a single matrix equation.
 
-	XXX Finish
+	For the one equation system, it is equivalent to
+	\[
+	    \mat{x-4y+z}=\mat{5}.
+	\]
+	Thus, we can rewrite \eqref{EQSYS1} as:
+	\[
+	    \mat{1&-4&1}\mat{x\\y\\z}=\mat{5}.
+	\]
```

Comment:
Let's reorder this explanation to something like: "We can rewrite \eqref{EQSYS1} as \[
	    \mat{1&-4&1}\mat{x\\y\\z}=\mat{5}.
	\]
Multiplying out to verify, we see,
\[
	    \mat{1&-4&1}\mat{x\\y\\z}=...=\mat{5},
	\]
which is indeed equivalent to \eqref{EQSYS1}"